src/mappings/mappingBalancesHandlers.ts

```diff
@@ -1,27 +1,84 @@
-import { SubstrateEvent } from "@subql/types";
-import { Balance } from "@polkadot/types/interfaces/runtime";
-import { checkIfExtrinsicExecuteSuccess } from "../helpers";
-import { BalanceTransfer } from "../types/models";
+import { SubstrateEvent } from '@subql/types'
+import { Balance } from '@polkadot/types/interfaces/runtime'
+import { BalanceTransfer, Rewards, TransferToTreasury } from '../types/models'
 
-export async function handleBalancesTransferEvent(event: SubstrateEvent) {
-    const from = event.event.data[0];
-    const to = event.event.data[1];
-    if(!from || !to) {
-        logger.error('Some of the from or to address is null', JSON.stringify(event.toHuman()));
-        return;
+const REWARD_ACCOUNTS = [
+    '4jbtsgNhpGAzdEGrKRb7g8Mq4ToNUpBVxeye942tWfG3gcYi',
+    '4jByf7kvkZ7hGYwGMYhjFYHoLec3zNZ3EKD86PARZDcfnnkD',
+    '1qnJN7FViy3HZaxZK9tGAA71zxHSBeUweirKqCaox4t8GT7',
+    '4hyBs59AiVKw4jZ851hmEVzpvxFATxwjvTDi67ziwb4vYDCX',
+    '4j5pigNy7LAX1dSGZQ7Tc2oms9dzZSEtwQJukSgQ9gAsQ9Fx',
```

Comment:
Can you remind where these addresses come from? you can also put a comment in front of each to remain here.